docker/opbeans/rum/tasks.js

```diff
@@ -3,32 +3,37 @@ const { Chromeless } = require('chromeless')
 var baseUrl = process.env.OPBEANS_BASE_URL || 'http://www.opbeans.com';
 var url = baseUrl
 
-function sleep(ms){
-    return new Promise(resolve=>{
-        setTimeout(resolve,ms)
+function sleep(ms) {
+    return new Promise(resolve => {
+        setTimeout(resolve, ms)
     })
 }
 
 async function run() {
-  const chromeless = new Chromeless({
-      lauchChrome: true
-  })
+    const chromeless = new Chromeless({
+        lauchChrome: true
+    })
+
+    for (; ;) {
+        url = await chromeless
+            .goto(url)
+            .evaluate((baseUrl) => {
+                if (typeof window.activateLoadGeneration === 'function') {
```

Comment:
maybe add a comment here that points the clueless reader (aka me) to https://github.com/elastic/opbeans-frontend/blob/7962b30a762f36c6b9c21b66970087ce9fa03035/src/rum.js#L28

Reply:
@beniwohli , I'm not sure about adding a reference to a specific line since it will be outdated quickly.

Reply:
But I think I should've provided that link in this PR's comment, I've updated the description

Reply:
@jahtalab didn't mean to make a comment to a specific line, more like "this function is defined in opbeans-frontend" or some such. I was stumped for a while because I couldn't find the function in this repo

Reply:
I thought since it's on `window` and inside the `chromless.evaluate` it should be clear it's in the frontend. But you can always be more clear 😄 . I've added the comment.

Thanks for the review!